pytest.ini

```diff
@@ -1,3 +1,8 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=ocfweb.settings
-addopts=--durations=10
+addopts=--durations=10 -n auto
```

Comment:
Probably want to document how people can lower this, as pytest will by default gobble up all of the available threads (which people may not like).

Reply:
I would vote to avoid duplicating pytest's documentation, but linking to it may be appropriate.

Reply:
Agreed, mentioning here that this will use all the available CPU threads (and linking to xdist's docs somewhere) is a good idea.